package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "porter",
+  "version": "1.0.0",
+  "dependencies": {
+    "aws-sdk": "^2.663.0",
+    "aws-xray-sdk": "^3.0.0",
+    "file-type": "^14.2.0",
+    "sharp": "^0.25.2"
+  },
+  "devDependencies": {
+    "eslint": "^6.8.0",
+    "eslint-config-airbnb-base": "^14.1.0",
+    "eslint-config-prettier": "^6.11.0",
+    "eslint-plugin-import": "^2.20.2",
+    "eslint-plugin-prettier": "^3.1.3",
+    "prettier": "^2.0.5"
+  },
+  "scripts": {
+    "eslint": "eslint",
+    "prettier": "prettier"
```

Comment:
No test script?

Oh, I see, the test directory has its own package.json.  Is that redundant now?  Seems like that should all move up to here.

Reply:
I don't look in the test directory anymore.